notebooks/COS/CalCOS/CalCOS.ipynb

```diff
@@ -8,39 +8,39 @@
     "\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    !calcos 
</pre><p>Please add an example path from this notebook demonstrating how this work in a comment. Thank you!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/hst_notebooks/pull/33/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>